src/ng/directive/select.js

```diff
@@ -183,15 +185,12 @@ var SelectController =
     } else if (interpolateValueFn) {
       // The value attribute is interpolated
       optionAttrs.$observe('value', function valueAttributeObserveAction(newVal) {
-        var currentVal = self.readValue();
```

Comment:
>
> Did this have side-effects?
>
It shouldn't have, this method is just checking some stuff & returning a
value.

-- 
Michał Gołębiowski